api/src/models/summaryresults-create.ts

```diff
@@ -0,0 +1,41 @@
+import { getLogger } from '../utils/logger';
+
+const defaultLog = getLogger('models/summaryresults-create');
```

Comment:
can we name this file `summary-results-create.ts` and then change this line to be:
```suggestion
const defaultLog = getLogger('models/summary-results-create');
```